packages/core/src/modules/routing/services/MediationRecipientService.ts

```diff
@@ -125,9 +134,9 @@ export class MediationRecipientService {
     // update keylist in mediationRecord
     for (const update of keylist) {
       if (update.action === KeylistUpdateAction.add) {
-        mediationRecord.addRecipientKey(update.recipientKey)
+        mediationRecord.addRecipientKey(verkeyToDidKey(update.recipientKey))
```

Comment:
Do we need a migration script to update non did keys?

Reply:
No, actually it was supposed to be a fix for the case where we receive keys in did:key format and it made it worst 😝.

It should use `didKeyToVerkey` instead. I'll add a test to `MediationRecipientService.test.ts` to make sure it stores all keys in base58 and it behaves exactly as current wallets.